plutus-tx/src/PlutusTx/Builtins/Internal.hs

```diff
@@ -679,3 +680,32 @@ bls12_381_mulMlResult (BuiltinBLS12_381_MlResult a) (BuiltinBLS12_381_MlResult b
 bls12_381_finalVerify ::  BuiltinBLS12_381_MlResult ->  BuiltinBLS12_381_MlResult -> BuiltinBool
 bls12_381_finalVerify (BuiltinBLS12_381_MlResult a) (BuiltinBLS12_381_MlResult b)
     = BuiltinBool $ BLS12_381.Pairing.finalVerify a b
+
+{-
+CONVERSION
+-}
+
+{-# NOINLINE builtinIntegerToByteString #-}
+builtinIntegerToByteString ::
+  BuiltinBool ->
+  BuiltinInteger ->
+  BuiltinInteger ->
+  BuiltinByteString
+builtinIntegerToByteString (BuiltinBool endiannessArg) paddingArg input =
+  case Convert.integerToByteStringWrapper endiannessArg paddingArg input of
+    Emitter f -> case runWriter f of
+      (result, logs) -> traceAll logs $ case result of
+        EvaluationFailure    -> mustBeReplaced "Integer to ByteString conversion errored."
```

Comment:
It could be useful for debugging purposes to include the input value in the error output.
Same for `builtinByteStringToInteger`

Reply:
It would be included as part of `traceAll logs`: see the definition of `integerToByStringWrapper` and what it emits.